src/Microsoft.ML.PCA/AssemblyInfo.cs

```diff
@@ -0,0 +1,8 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.CompilerServices;
+using Microsoft.ML;
+
+[assembly: InternalsVisibleTo(assemblyName: "Microsoft.ML.Tests" + PublicKey.TestValue)]
```

Comment:
>PublicKey [](start = 67, length = 9)

if you change PcaTests.cs to use ml.Transforms.Projection.ProjectToPrincipalComponents instead of direct estimator call would you still need this change?